### DISCUSSION TITLE
Change border/background color for comic book pages

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#4131] [General]
Hello. Is it possible to change these black borders while reading comic book (CBZ) files? I tried editing MainWindowBackground, but it didn't change it. 

![example](https://github.com/sumatrapdfreader/sumatrapdf/assets/151783497/8b51e187-c292-4820-8e38-bba2b30ca04d)

**Comment by GitHubRulesOK** [comment]
Currently no as images like many graphics apps are traditionally shown on a dark (presentation) background.

**Reply by ghost** [reply]
I see. Would that be too complicated for you to implement? I actually have a collection of comic books with cropped pages (no borders around the strips), so it'd be nice to set a light background for them.

**Reply by GitHubRulesOK** [reply]
I am support not development but the request was made sometime ago
however there is only one part time code owner/developer
related to https://github.com/sumatrapdfreader/sumatrapdf/issues/1069 and https://github.com/sumatrapdfreader/sumatrapdf/issues/853

**Reply by ghost** [reply]
Thank you for the links. For some reason I didn't see them when I looked "cbz" up.
